features/scaffold-package.feature

```diff
@@ -121,40 +156,61 @@ Feature: Scaffold WP-CLI commands
       """
       All package files were skipped
       """
+    When I run `wp package uninstall wp-cli/same-package`
+    Then STDOUT should contain:
+      """
+      Success: Uninstalled package.
+      """
 
   Scenario: Scaffold a WP-CLI command with tests
     Given an empty directory
 
-    When I run `WP_CLI_PACKAGES_DIR=packages wp scaffold package wp-cli/foo`
+    When I run `wp package path`
+    Then save STDOUT as {PACKAGE_PATH}
+
+    When I run `wp scaffold package wp-cli/with-tests`
     Then STDOUT should contain:
       """
       Success: Created package files
       """
-    And the packages/vendor/wp-cli/foo/.gitignore file should exist
-    And the packages/vendor/wp-cli/foo/.editorconfig file should exist
-    And the packages/vendor/wp-cli/foo/composer.json file should exist
-    And the packages/vendor/wp-cli/foo/command.php file should exist
-    And the packages/vendor/wp-cli/foo/wp-cli.yml file should exist
-    And the packages/vendor/wp-cli/foo/.travis.yml file should exist
-    And the packages/vendor/wp-cli/foo/features/bootstrap/Process.php file should exist
-    And the packages/vendor/wp-cli/foo/features/bootstrap/ProcessRun.php file should exist
+    And the {PACKAGE_PATH}/local/wp-cli/with-tests/.gitignore file should exist
+    And the {PACKAGE_PATH}/local/wp-cli/with-tests/.editorconfig file should exist
+    And the {PACKAGE_PATH}/local/wp-cli/with-tests/composer.json file should exist
+    And the {PACKAGE_PATH}/local/wp-cli/with-tests/command.php file should exist
+    And the {PACKAGE_PATH}/local/wp-cli/with-tests/wp-cli.yml file should exist
+    And the {PACKAGE_PATH}/local/wp-cli/with-tests/.travis.yml file should exist
+    And the {PACKAGE_PATH}/local/wp-cli/with-tests/features/bootstrap/Process.php file should exist
+    And the {PACKAGE_PATH}/local/wp-cli/with-tests/features/bootstrap/ProcessRun.php file should exist
 
-    When I run `WP_CLI_PACKAGES_DIR=packages wp --require=packages/vendor/wp-cli/foo/command.php hello-world`
+    When I run `wp --require={PACKAGE_PATH}/local/wp-cli/with-tests/command.php hello-world`
```

Comment:
> Let's go ahead and create a new one so we have coverage in both directions.

I have created a new scenario without `--require` and left this one in place as it was. Or would you prefer I adjust the existing tests?

The new test would probably become obsolete in that case.